app/views/partials/_job_listing.html.erb

```diff
@@ -1,6 +1,18 @@
 <h2 class="job-position"><%= @job.position%></h2>
 <h4>Posting Date: <%= time_tag(@job.posting_date) %><h4>
-<p><a href="#" id="company-link"><%= @job.company.name %></a> <%= @job.location %></p>
+<% if @job.company.url.present? %>
+  <p><%= link_to @job.company.name, @job.company.url %> <%= @job.location %></p>
+<% else %>
+  <p><%= @job.company.name %> <%= @job.location %></p>
+<% end %>
+<div class="tags">
+  <% @job.tags.each do |tag| %>
+    <div class="tag">
+      <%= tag.name %>
+    </div>
+  <% end %>
+</div>
 <div class="job-description">
   <h3 class="job-description-header">Job Description</h3>
-  <p><%= @job_description %></p>
+  <p class="job-description-p"><%= @job_description %></p>
+</div>
```

Comment:
@EmilyMB brandon said that he was waiting on something you were doing to make sure that the company url was formatted right with http://...is that ready? can this be merged based on the backend?